src/components/Shortcuts/Shortcut.js

```diff
@@ -9,7 +9,7 @@ const Shortcut = (item, _key) => {
   if (Array.isArray(item)) {
     return (
       <li key={_key}>
-        <Shortcuts data={item} />
+        <Shortcuts className="-is-logged-in" data={item} />
```

Comment:
@ceanela will this add -is-logged-in to the ul list container or the li's themselves?

Reply:
This just adds it to the container

Reply:
Perfect